pkg/storage/stores/shipper/compactor/retention/marker.go

```diff
@@ -221,6 +220,29 @@ func (r *markerProcessor) Start(deleteFunc func(ctx context.Context, chunkId []b
 
 		}
 	}()
+	r.wg.Add(1)
+	go func() {
+		defer r.wg.Done()
+		ticker := time.NewTicker(10 * time.Second)
```

Comment:
Why every 10s and not more frequently?

Reply:
Most scrape configs for prometheus are 15s, which gives good freshness I think ? I'm not close to increase that if you have reason for it.

Reply:
I do not have a strong preference, but 10s looks like a sweet spot with most people doing 15s or higher scrape intervals, and it should be okay to see the metric getting updated a little late.

Reply:
Cool, happy with that :+1: